Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -671,6 +671,12 @@ void Painter::draw_bitmap(IntPoint const& p, GlyphBitmap const& bitmap, Color co
     }
 }
 
+void Painter::draw_triangle(IntPoint offset, AK::Vector<IntPoint> const& control_points, Color color)
```

Comment:
`control_points` should be a `Span<const IntPoint> rather than `AK::Vector<IntPoint> const&` there's no reason for this to be specific to `Vector`.